lib/ContractInteract/BrandedToken.js

```diff
@@ -0,0 +1,393 @@
+    this.acceptStakeRequestRawTx = this.acceptStakeRequestRawTx.bind(this);
+    this.liftRestriction = this.liftRestriction.bind(this);
+    this.liftRestrictionRawTx = this.liftRestrictionRawTx.bind(this);
+    this.isUnrestricted = this.isUnrestricted.bind(this);
+    this.rejectStakeRequest = this.rejectStakeRequest.bind(this);
+    this.rejectStakeRequestRawTx = this.rejectStakeRequestRawTx.bind(this);
+  }
+
+  /**
+   * Deploys a Branded token contract.
+   *
+   * @param web3 Origin chain web3 object.
+   * @param valueToken The value to which valueToken is set.
+   * @param symbol The value to which tokenSymbol, defined in EIP20Token,
+   *                is set.
+   * @param name The value to which tokenName, defined in EIP20Token,
+   *              is set.
+   * @param decimals The value to which tokenDecimals, defined in EIP20Token,
+   *                  is set.
+   * @param conversionRate The value to which conversionRate is set.
+   * @param conversionRateDecimals The value to which
+   *                                conversionRateDecimals is set.
+   * @param organization Organization contract address.
+   * @param {Object} txOptions Transaction options.
+   *
+   * @returns {Promise<BrandedToken>} Promise containing the Branded token
+   *                                  instance that has been deployed.
+   */
+  static async deploy(
+    web3,
+    valueToken,
+    symbol,
+    name,
+    decimals,
+    conversionRate,
+    conversionRateDecimals,
+    organization,
+    txOptions,
+  ) {
+    if (!txOptions) {
+      const err = new TypeError('Invalid transaction options.');
+      return Promise.reject(err);
+    }
+    if (!Web3.utils.isAddress(txOptions.from)) {
+      const err = new TypeError(`Invalid from address: ${txOptions.from}.`);
+      return Promise.reject(err);
+    }
+
+    const tx = BrandedToken.deployRawTx(
+      web3,
+      valueToken,
+      symbol,
+      name,
+      decimals,
+      conversionRate,
+      conversionRateDecimals,
+      organization,
+    );
+
+    return Utils.sendTransaction(tx, txOptions).then((txReceipt) => {
+      const address = txReceipt.contractAddress;
+      return new BrandedToken(web3, address);
+    });
+  }
+
+  /**
+   * Raw transaction for {@link BrandedToken#deploy}.
+   *
+   * @param web3 Origin chain web3 object.
+   * @param valueToken The value to which valueToken is set.
+   * @param symbol The value to which tokenSymbol, defined in EIP20Token,
+   *                is set.
+   * @param name The value to which tokenName, defined in EIP20Token,
+   *              is set.
+   * @param decimals The value to which tokenDecimals, defined in EIP20Token,
+   *                  is set.
+   * @param conversionRate The value to which conversionRate is set.
+   * @param conversionRateDecimals The value to which
+   *                                conversionRateDecimals is set.
+   * @param organization Organization contract address.
+   *
```

Comment:
"A value to which ... is set" is not a very valuable documentation. Instead, it should explain what the parameters are for/explain them.

Reply:
I can add a @dev note in the document explaining conversion rate and decimal. 
Does this work for you?

Reply:
I don't find `@dev` as a valid JSDoc annotation. It would also not be correct as this info is not intended for developers, but for consumers of the function and should therefore not be hidden behind `@dev`.
Why don't you explain the parameters where they're supposed to be explained?